packages/evolution-generator/src/scripts/generate_survey.py

```diff
@@ -33,7 +34,28 @@ def generate_survey(config_path):
         # Get the data from the YAML file
         survey_folder_path = surveyGenerator["survey_folder_path"]
         excel_file_path = surveyGenerator["excel_file_path"]
-        enabled_scripts = surveyGenerator["enabled_scripts"]
+        enabled_scripts = surveyGenerator.get("enabled_scripts", [])
+
+        required_scripts = [
```

Comment:
Required by who? and/or for what?

Reply:
If there were no scripts, I had an error, but without much information if you don't know what was going on. I might propose a better way of doing that. I could probably just send a default value of false, and there won't be any error.